frontend/src/pages/IntegrationAuthCallback/IntegrationAuthCallbackPage.tsx

```diff
@@ -266,7 +267,7 @@ const WorkspaceIntegrationCallback = ({
 
 	useEffect(() => {
 		if (!addIntegration || !code) return
-		const next = `/${projectId}/integrations/${type}`
+		const redirectUrl = next || `/${projectId}/integrations/${type}`
```

Comment:
Only GitHub is passing next in here, so ClickUp and Height will still use the fallback